thumbor_cloud_storage/result_storages/cloud_storage.py

```diff
@@ -74,7 +74,7 @@ def get(self):
         return blob.download_as_string()
 
     def normalize_path(self, path):
-        path_segments = [self.context.config.get('RESULT_STORAGE_CLOUD_STORAGE_ROOT_PATH','thumbor/').rstrip('/'), Storage.PATH_FORMAT_VERSION, ]
+        path_segments = [self.context.config.get('RESULT_STORAGE_CLOUD_STORAGE_ROOT_PATH','thumbor/result_storage').rstrip('/'), Storage.PATH_FORMAT_VERSION, ]
```

Comment:
This will be a backwards incompatible change, won't it? Which could incur a bunch of cache misses and costs to those upgrading.

I think it'd be better to recommend this and leave the result storage where it is, and then move the other storage into a subdirectory or different root directory, e.g. `thumbor-strorage`, with a recommendation to move them into better named directories or sub-directories.